2019-02-10 10:19:41,050 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:21:54,099 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:25:23,659 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:42:04,727 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:48:21,200 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:58:12,223 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:58:17,209 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:59:23,084 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:59:27,503 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 10:59:48,428 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 11:01:14,148 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 11:01:24,212 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 11:04:12,436 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 11:04:24,418 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 11:04:30,363 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:33:26,378 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:33:55,170 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:34:00,815 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:35:55,354 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:36:01,904 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:36:25,906 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:36:35,154 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:41:06,076 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:42:35,757 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:43:32,737 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-10 19:50:00,116 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-26 07:17:23,967 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:73]
2019-02-26 07:25:46,270 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:76]
2019-02-26 07:27:03,275 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:76]
2019-02-26 07:27:06,732 ERROR: Exception on /project [GET] [in /home/holger/.local/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/holger/git/labHandle/app/project/routes.py", line 11, in projects
    return render_template('projects.view', title='')
  File "/home/holger/.local/lib/python3.5/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: projects.view
2019-02-26 07:27:45,855 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:76]
2019-02-26 07:27:50,527 ERROR: Exception on /project [GET] [in /home/holger/.local/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/holger/.local/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/holger/git/labHandle/app/project/routes.py", line 11, in projects
    return render_template('project.project', title='')
  File "/home/holger/.local/lib/python3.5/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/holger/.local/lib/python3.5/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/holger/.local/lib/python3.5/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: project.project
2019-02-26 07:28:25,429 INFO: Length dep. server startup [in /home/holger/git/labHandle/app/__init__.py:76]
